<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>289</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(225, 225, 225); font-size: 14pt</string>
   </property>
   <property name="text">
    <string>Departure</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="departure">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 12pt; 
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>571</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 20pt; 
color: rgb(227, 227, 227);</string>
   </property>
   <property name="text">
    <string>Enter the city of departure and the city of arrival</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submin_cities_btn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(227, 227, 227); font-size: 14pt;
background-color: rgb(113, 113, 113);</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(225, 225, 225); font-size: 14pt</string>
   </property>
   <property name="text">
    <string>Arrival</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="arrival">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 12pt; 
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
